components/notability/commons/notability_checker.lua

```diff
@@ -17,7 +17,7 @@ local NotabilityChecker = {}
 
 local _lang = mw.language.new('en')
 local _NOW = os.time()
-local _SECONDS_IN_YEAR = 365.25 * 86400
+local _SECONDS_IN_YEAR = 365.2425 * 86400
```

Comment:
🤔

Reply:
actually that number is correct^^
365 days in most years
every 4th year 1 more day —> +0.25
but not if devisable by 100 —> -0.01
unless it is devisable by 400 —> +0.0025
total 365.2425

Reply:
I wasn't doubting it 😉